src/terms/pairwise.jl

```diff
@@ -94,7 +84,7 @@ function energy_pairwise(lattice, symbols, positions, V, params;
     Rlims = estimate_integer_lattice_bounds(lattice, max_radius, poslims)
 
     # Check if some coordinates are not used.
-    is_dim_trivial = [norm(lattice[:,i]) == 0 for i=1:3]
+    is_dim_trivial = [iszero(norm(lattice[:,i])) for i=1:3]
```

Comment:
no need for the norm( ), could just do `iszero(lattice[:, i])` (also missing space btw.)